run-tests.sh

```diff
@@ -6,9 +6,7 @@
 #
 # Invenio is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
-
-pydocstyle invenio_accounts tests docs && \
-isort -rc -c -df && \
+docker-services-cli up && \
```

Comment:
probably you don't need this. You can have this file as in the module cookiecutter, without `docker-services-cli`:

```
python -m check_manifest --ignore ".travis-*" && \
python -m sphinx.cmd.build -qnNW docs docs/_build/html && \
python -m pytest
python -m sphinx.cmd.build -qnNW -b doctest docs docs/_build/doctest
tests_exit_code=$?
exit "$tests_exit_code"
```